compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.td

```diff
@@ -82,8 +84,9 @@ def DispatchLoweringPassPipelineEnum : I32EnumAttr<
     // LLVMGPU CodeGen pipelines
     LLVMGPU_Default, LLVMGPU_SimpleDistribute, LLVMGPU_Vectorize,
     LLVMGPU_MatmulSimt, LLVMGPU_MatmulTensorCore,
-    LLVMGPU_TransposeSharedMem, LLVMGPU_WarpReduction,
-    LLVMGPU_PackUnPack, LLVMGPU_MatmulTensorCoreMmaSync,
+    LLVMGPU_TransposeSharedMem, LLVMGPU_VectorDistribute,
```

Comment:
Could we organize these to be alphabetical?

Reply:
I looked and none of the other backends are alphabetical. My best guess is that when we add a new pipeline, we just increment the enum so as to not break existing .mlirbc files (if that's worth anything). I think all the pipelines here could use a cleanup so for now I just reordered the new pipeline in this PR to match the enum ordering (to be consistent with other backends).